docs/source/cli_controller.md

```diff
@@ -72,4 +72,23 @@ Deleted task with ID(s): ['123', '456']
 >>> from todo_cli.cli.cli_controller import delete_many
 >>> delete_many('123')
 Please provide at least 2 task id's to delete.
-``` 
\ No newline at end of file
+``` 
+
+## Export
+
+The Export function allows you to export your data. There are 2 options when it comes to exporting:
+Exporting to JSON file:
+
+```python
+>>> from todo_cli.cli.cli_controller import export
+>>> export json file_name.json
```

Comment:
Since we are referring to a code context here, we should document it like this:

```python
>>> export("json", "file_name.json")
```

We can also document it like this:

```python
>>> from todo_cli.cli.cli_controller import export
>>> from todo_cli.cli.cli_controller import Formats
>>> export(Formats.JSON, "exported.json")